core-ui/src/components/Extensibility/hooks/useJsonata.js

```diff
@@ -5,37 +5,44 @@ import { last, mapValues } from 'lodash';
 import { jsonataWrapper } from '../helpers/jsonataWrapper';
 import { DataSourcesContext } from '../contexts/DataSources';
 
-export function useJsonata(
```

Comment:
Could we add types here? I can see that we return a function that returns an array of 2 or sometimes 3 elements and also has `async` property, it would be easier to use if we had this stated explicitly.